website/static/schemas/next/chatMessage.schema.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/chatMessage.schema.json",
+  "type": "object",
+  "title": "ChatMessage",
+  "allOf": [{ "$ref": "context.schema.json#" }],
+  "properties": {
+    "type": { "const": "fdc3.chat.message" },
+    "chatRoom": { "$ref": "chatRoom.schema.json#" },
+    "message": { "$ref": "message.schema.json#" }
```

Comment:
It uses a schema introduced in another PR https://github.com/finos/FDC3/pull/779 
(See [schema](https://github.com/finos/FDC3/blob/75141ff3561c22399b4b7dd34795d35f5f1307c5/src/context/schemas/message.schema.json) )